portfolio/src/main/webapp/script.js

```diff
@@ -45,9 +45,9 @@ function moreFacts(){
   document.getElementById("displayFact").innerHTML= facts[rand];
   document.getElementById("displayFact").style.border="double";
 }
-async function getName(){
-    const response = await fetch('/data');
-    const name = await response.text();
-    document.getElementById('showName').innerHTML=name;
-
+async function getComments(){
+  const response = await fetch('/data');
+  const comments = await response.text();
+  console.log(comments);
```

Comment:
Do you want to print this here?

Reply:
The console.log is no longer there I removed it in the last commit i did (style the comments)